nextflow_schema.json

```diff
@@ -50,96 +49,110 @@
         "main_options": {
             "title": "Main options",
             "type": "object",
-            "description": "Option used for most of the pipeline.",
+            "description": "Most common options used for the pipeline",
             "default": "",
             "properties": {
-                "tools": {
-                    "type": "string",
-                    "fa_icon": "fas fa-toolbox",
-                    "description": "Tools to use for variant calling and/or for annotation.",
-                    "help_text": "Multiple separated with commas.\n\nGermline variant calling can currently only be performed with the following variant callers:\n- FreeBayes, HaplotypeCaller, Manta, mpileup, Strelka, TIDDIT\n\nSomatic variant calling can currently only be performed with the following variant callers:\n- ASCAT, Control-FREEC, FreeBayes, Manta, MSIsensorpro, Mutect2, Strelka\n\n**NB** Mutect2 for somatic variant calling cannot be combined with `--no_intervals`\n\nTumor-only somatic variant calling can currently only be performed with the following variant callers:\n- Control-FREEC, Manta, mpileup, Mutect2, TIDDIT\n\nAnnotation is done using snpEff, VEP, or even both consecutively.\n\n> **NB** As Sarek will use bgzip and tabix to compress and index VCF files annotated, it expects VCF files to be sorted.\n\n> **NB** tools can be specified with no concern for case.",
-                    "pattern": "^((ascat|cnvkit|controlfreec|deepvariant|dnascope|dnaseq|freebayes|haplotypecaller|manta|merge|mpileup|msisensorpro|mutect2|snpeff|strelka|tiddit|tnscope|vep)*,?)*$"
+                "split_fastq": {
+                    "type": "integer",
+                    "default": 0,
+                    "fa_icon": "fas fa-clock",
+                    "description": "Specify how many reads each split of a FastQ file contains. Set 0 to turn of splitting at all.",
```

Comment:
Wouldn't it make sense to have that one back in the `fastq_preprocessing` section?

Reply:
I am a bit conflicted about it. On the one hand it does something to the fastq files, on the other hand  it is used for speed up but doesn't change the reads themselves.

Reply:
Makes sense.